src/NLog.AWS.Logger/AWSTarget.cs

```diff
@@ -29,7 +29,6 @@ public class AWSTarget : TargetWithLayout, IAWSLoggerConfig
         /// </summary>
         public AWSTarget()
         {
-            this.OptimizeBufferReuse = true;
```

Comment:
whats the reason for this change?

Reply:
`OptimizeBufferReuse` was introduced in NLog v4 as opt-in optimization. With NLog v5 then it is always enabled, and the setting has been marked as obsolete.